index.js

```diff
@@ -80,6 +80,28 @@ const parseInput = async input => {
 	return input;
 };
 
+const killChecker = async input => {
+	let i = 0;
+
+	return new Promise(resolve => {
+		const checker = setInterval(async () => {
+			i++;
+			const exists = await processExists(input);
```

Comment:
If `processExists` rejects, it will cause an uncaught exception as it's not caught anywhere, since you're using `new Promise`.